src/opcodes.js

```diff
@@ -0,0 +1,370 @@
+module.exports = new Map([
+  // push value
+  ['OP_0', function (context) {
```

Comment:
Should the keys be the ASM term or the opcode byte?

Reply:
really doesn't matter, but I think it makes the `opcodes.js` file more readable if they are words.